pkgs/tools/misc/fselect/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "fselect";
+  version = "0.6.6";
+
+  src = fetchFromGitHub {
+    owner = "jhspetersson";
+    repo = "fselect";
+    rev = version;
+    sha256 = "0lwh0qcyy9hmkxijlhp0inznrb0ibw1wcnnlqfxc71y4ar42f165";
+  };
+
+  cargoSha256 = "0s7yld3cwmyfdsqwcipc2l07xx5qn3knjmh4bbnjmnxiygaiszx9";
+
+  cargoPatches = [ ./cargo-lock.patch ];
+
+  meta = with stdenv.lib; {
+    description = "Find files with SQL-like queries";
+    homepage = "https://github.com/jhspetersson/fselect";
+    license = with licenses; [ asl20 /* or */ mit ];
+    maintainers = [ ];
```

Comment:
can you add yourself as the maintainer?

Reply:
Would be great, but I already have a PR adding me to the maintainer list (for `ruplacer`). Is it okay for me to duplicate these changes here?